src/main/java/org/myrobotlab/service/InMoovHand.java

```diff
@@ -591,7 +591,7 @@ static public ServiceType getMetaData() {
     meta.addPeer("pinky", "Servo", "Pinky servo");
     meta.addPeer("wrist", "Servo", "Wrist servo");
     meta.addPeer("arduino", "Arduino", "Arduino controller for this arm");
-    meta.addPeer("leap", "LeapMotion", "Leap Motion Service");
+    meta.addPeer("leap", "LeapMotion", "Leap Motion Service", false);

```

Comment:
instead of adding this additinal boolean to addPeer, i'd prefer to see the leap motion service get fixed so it doesn't kill the jvm.  we should consider removal of this additional boolean here..

Reply:
Peer lifecycle is relatively new.. I was willing to give it flexibility.
We could certainly remove it, and the more strict life-cycle definition would make me happy.

But, I wanted to keep a card up my sleeve if I had to play it.
The killing of the jvm is a static load and exit on Leap - its just "poor code design"
If we have the correct natives, and reference to the java.library.path - all is happy.
But, personally, I'd prefer looking at a log file and seeing it error vs. someone just describing, "MRL just doesnt start"